agnostic-vue/src/components/ChoiceInput.vue

```diff
@@ -194,7 +194,13 @@ const triggerChange = (e) => {
   const value = e.target.value;
   if (checked) {
     if (!mutableCheckedOptions.includes(value)) {
-      mutableCheckedOptions.push(value);
+      if (props.type === "radio") {
```

Comment:
This fixed a bug where radio event was just pushing values onto an array (should only ever be one checked radio)